src/shared/scrapers/USA/VI/index.js

```diff
@@ -0,0 +1,51 @@
+import * as fetch from '../../../lib/fetch/index.js';
+import * as parse from '../../../lib/parse.js';
+
+const scraper = {
+  state: 'VI',
+  country: 'USA', // The alpha-3 country code is VIR
```

Comment:
Hmm, then do we need to have `VIR` here instead?

Reply:
Your call - I tried a few different combinations of country/state/featureId but couldn't find one that was matched during the update task.